bin/qtui-rebind

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+"""
+    Simple script to rewrite Qt ui files to change shortcut bindings.
```

Comment:
I don't think that's Eco's current style (for better or worse), so I suggest we follow the project's existing precedent. [Personally, I would probably make this a comment rather than a docstring, but maybe that's just me.]